src/Pages/Adopt.jsx

```diff
@@ -217,247 +288,406 @@ export const Adopt = () => {
+                  render={({ field: { onChange, value, name } }) => (
+                    <RadioGroup
+                      aria-labelledby="other-birds-in-household-bool"
+                      value={value}
+                      name={name}
+                      onChange={(evt) => {
+                        clearErrors([
+                          "other_bird_species",
+                          "other_bird_checkup_date",
+                          "other_bird_diet",
+                        ]);
+                        onChange(evt);
+                      }}
+                    >
+                      <FormControlLabel
+                        value="Yes"
+                        control={<Radio />}
+                        label="Yes"
+                      />
+                      <FormControlLabel
+                        value="No"
+                        control={<Radio />}
+                        label="No"
+                      />
+                    </RadioGroup>
+                  )}
                 />
-                <FormControlLabel
-                  value="shared-housing"
-                  control={<Radio />}
-                  label="Shared Housing"
+                {watchOtherBirds === "Yes" && (
+                  <Fade
+                    in={true}
+                    timeout={250}
+                  >
+                    <Stack spacing={2}>
+                      <TextField
+                        id="other-birds-in-household-text"
+                        label="If yes, list species and how many"
+                        variant="outlined"
+                        multiline
+                        minRows={4}
+                        {...register("other_bird_species", {
+                          required: "Other bird species is required",
+                        })}
+                        error={!!errors.other_bird_species?.message}
+                        helperText={errors.other_bird_species?.message}
+                      />
+                      <TextField
+                        id="last-bird-checkup-date"
+                        label="If you currently have birds, what was the date of their last annual checkup?"
+                        variant="outlined"
+                        {...register("other_bird_checkup_date", {
+                          required: "Last checkup date is required",
+                        })}
+                        error={!!errors.other_bird_checkup_date?.message}
+                        helperText={errors.other_bird_checkup_date?.message}
+                      />
+                      <TextField
+                        id="bird-diet"
+                        label="List the current diet that you feed your birds"
+                        variant="outlined"
+                        multiline
+                        minRows={4}
+                        {...register("other_bird_diet", {
+                          required: "Other bird diet is required",
```

Comment:
Let's change this error message to something like "Current diet is required"

Reply:
Sounds good!